tmail-backend/rate-limiter/rate-limiter-cassandra/src/main/scala/com/linagora/tmail/rate/limiter/cassandra/CassandraRateLimitationPlanRepository.scala

```diff
@@ -29,7 +28,7 @@ class CassandraRateLimitationPlanRepository @Inject()(cassandraRateLimitPlanDAO:
       .`then`(create(resetRequest.id, RateLimitingPlanEntry.from(resetRequest)))
       .`then`()
 
-  override def planExists(id: RateLimitingPlanId): Publisher[Boolean] = cassandraRateLimitPlanDAO.planExists(id)
+  override def planExists(id: RateLimitingPlanId): Publisher[java.lang.Boolean] = cassandraRateLimitPlanDAO.planExists(id).map(boolean2Boolean)
```

Comment:
It seems the function def in the trait has that signature. I find that strange, I think it should stay in Scala and the java class using it should do the conversion... but ok!